gnat2goto/driver/arrays.adb

```diff
@@ -258,6 +258,58 @@ package body Arrays is
          Append_Struct_Member (Array_Temp_Struct, High_Expr);
          Append_Struct_Member (Array_Temp_Struct, Malloc_Call_Expr);
 
+         if Present (Component_Associations (N)) and then
+           List_Length (Component_Associations (N)) /= 1
+         then
+            declare
+               Components : constant List_Id := Component_Associations (N);
+               Component_Node : Node_Id := First (Components);
+            begin
+               if List_Length (Choices (Component_Node)) /= 1 then
+                  return Report_Unhandled_Node_Irep (N,
+                                     "Do_Aggregate_Literal_Array",
+                                     "More than one choice in component node");
```

Comment:
what does this mean? Can we document this here? (I'm assuming this is for the `(A|B|C => Val1, E|F|G => Val2)` syntax)

(I don't think this has to be implemented as part of this PR, but looks like it could be adapted reasonably easily?)